README.md

```diff
@@ -1,79 +1,86 @@
+
 # BreezewayMod
+
 ## About
-![](breezewaymod_icon.png) 
 
-BreezewayMod is a fork of the Breezeway window decoration which aims to mimic the macOS interface. But with powerups.
-Did I mention that it comes without weird glitches on the rounded corners?
+<img src="breezewaymod_icon.png" style="zoom: 25%;" /> 
 
-## Screenshots
-![](screenshot.png)
+**BreezewayMod** is a fork of the [Breezeway](https://github.com/richard-kung/breezeway) window decoration, which aims to mimic the *macOS style*, but with **power ups**.
 
-Looks good with light and dark color schemes (shown here with color schemes "Arc" for light windows and "Arc-Dark" for dark windows)
+> **Note**: it comes without weird glitches on the rounded corners.
 
-![](sierrabreeze.png)
+## Preview
 
-The usual glitch on dark themes.
+![](screenshot.png)
 
-![](breezewaymod.png)
+> Looks great  with light and dark color schemes. Shown here with [Arc](https://github.com/PapirusDevelopmentTeam/arc-kde) and [Arc-Dark](https://github.com/PapirusDevelopmentTeam/arc-kde) color schemes.
 
-Glitch-b-gone with Breezeway magick!
 
-![](low_contrast.png)
+| The usual glitch on dark themes            | ![](sierrabreeze.png) |
+| ------------------------------------------ | :-------------------: |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](breezewaymod.png) |
+| Glitch be gone with BreezewayMod. *Magic!* | ![](low_contrast.png) |
```

Comment:
In general I'm happy with this, however the last entry in that table has duplicate text. The screenshot shows how windows create stronger outlines in low-contrast environments.

Reply:
Done.